.mypy.ini

```diff
@@ -0,0 +1,26 @@
+[mypy]
+files = tests/test_multidict.py, tests/test_mypy.py
+check_untyped_defs = True
+disable_error_code = comparison-overlap  # Causes comparison errors with Dict.
```

Comment:
Not sure if there is something we can do to improve this (maybe inherit from `dict`?). But, throughout the tests I get errors when comparing with dicts, like: `Non-overlapping equality check (left operand type: "Union[MultiDict[str], CIMultiDict[str]]", right operand type: "Dict[str, str]")`

Reply:
@webknjaz Not sure if you want to try addressing this in the next release or not? I'll just continue ignoring it for now.